src/proxyServer/state.js

```diff
@@ -1,22 +1,26 @@
 const fs = require('fs')
 const path = require('path')
 const branchName = require('branch-name')
+const CONFIG = require('../constants/config')
 
 let state = {
   branchName: '',
   browser: 'chrome',
   cachingRef: '',
-  certAuthority: {
-    key: fs.readFileSync(
-      path.resolve(__dirname, '../assets/ssl/proxypack.key.pem'),
-    ),
-    cert: fs.readFileSync(
-      path.resolve(__dirname, '../assets/ssl/proxypack.crt.pem'),
-    ),
+  cert: {
+    ca: {
+      cert: fs.readFileSync(path.resolve(__dirname, CONFIG.SSL_CERTS.CA)),
+      key: fs.readFileSync(path.resolve(__dirname, CONFIG.SSL_CERTS.CA_KEY)),
+    },
+    server: {
+      cert: fs.readFileSync(path.resolve(__dirname, CONFIG.SSL_CERTS.SERVER)),
+      key: fs.readFileSync(
+        path.resolve(__dirname, CONFIG.SSL_CERTS.SERVER_KEY),
+      ),
+    },
```

Comment:
Yes we need the CA and the Server Cert, that is the big break through.

SSL certs are just like a big pyramid scheme.

Reply:
Basically alot of my problems came from trying to use the CA on a node server. But in fact you just want to use the CA to sign a cert for that server.

Reply:
http://c.hlp.sc/2f686bacd8e6